src/controller/python/chip/yaml/format_converter.py

```diff
@@ -23,8 +23,49 @@
 import binascii
 
 
+def _substitute_in_config_variables(field_value, config_values: dict):
+    ''' Substitutes values that are config variables.
+
+    YAML values can contain a string of a configuration variable name. In these instances we
+    substitute the configuration variable name with the actual value.
+
+    # TODO This should also substitue any saveAs values as well as perform any required
+    # evaluations.
+
+    Args:
+        'field_value': Value as extracted from YAML.
+        'config_values': Dictionary of global configuration variables.
+    Returns:
+        Value with all global configuration variables substituted with the real value.
+    '''
+    if (type(field_value) is dict):
+        return_values = {}
+        for key in field_value:
+            return_values[key] = _substitute_in_config_variables(field_value[key], config_values)
+
+        return return_values
+    elif(type(field_value) is list):
```

Comment:
C++-style parenthesizing should be removed here.

Also see previous comment re: detecting sequence